ADAL/src/cache/ADTokenCacheAccessor.h

```diff
@@ -20,6 +20,7 @@
 
 @class ADTokenCacheKey;
 @class ADTokenCacheItem;
+@class ADAuthenticationError;
```

Comment:
When I tried to include ADTokenCacheAccessor.h, I got some errors asking for forward declaration :P